src/renderer.js

```diff
@@ -80,7 +80,7 @@ export function toArray(text, options = {}) {
         }
 
         const isMaybePartOfBiggerAlias =
-          maybeBiggerAliasCharacters !== undefined;
+          maybeBiggerAliasCharacters !== undefined && fullMatch[0] == ":";
```

Comment:
Why do you need this ? After removing it tests still pass.

Reply:
The code assumes later on that the first character of the full match is a `:` when it checks whether `.slice(1, -1)` is an alias (e.g. `:o` and `:open_mouth:`). There are however several ascii aliases that don't start with `:`, like `<3`.

It does turn out that I made a mistake in checking.
Previously, `<3:` would not be replaced at all, this version replaces it with `❤️`, but we want `❤️:`.

If you put the check later it works:

```javascript
if (fullMatch[0] == ":" && isPartOfBiggerAlias) {
```

Reply:
Maybe fix it that way and add a test?

Reply:
I see, yes definitely ! 🙏 Make triple equals thought 😉 (`fullMatch[0] === ":`)

(answer continues in next comment)